### PR TITLE
Switch to using MySQL DB Driver

### PR DESCRIPTION
**Pull request by joshmcfarlane**
Resolves https://github.com/ColectivaLegal/ScalaSmsAlertSystem/issues/2

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -375,6 +375,15 @@ db {
   #default.logSql=true
 }
 
-slick.dbs.default.profile = "slick.jdbc.H2Profile$"
-slick.dbs.default.db.profile="org.h2.Driver"
-slick.dbs.default.db.url="jdbc:h2:mem:play;DB_CLOSE_DELAY=-1"
\ No newline at end of file
+# Uncomment for testing locally. May require slight tweaks to evolutoins to support H2 schema.
```
Comment (ErrorsAndGlitches) [inline conf/application.conf]: I was reading through:
https://www.playframework.com/documentation/2.6.x/ProductionConfiguration

Would it be possible to have a `prod.conf` and `develop.conf` which includes the common configuration, which we can put in `common.conf`? HOCON contains [includes](https://github.com/lightbend/config/blob/master/HOCON.md#includes), which could include the `common.conf` in the other two files. We could then pass `-Dconfig.resource=prod.conf` for example in the CLI.

Comment (joshmcfarlane) [reply conf/application.conf]: Ah was waiting on the review but somehow missed this. 

We could separate out the configuration, and I was actually planning on doing it in a later CR, but it doesn't quite solve the evolution issue - There doesn't seem to be a good union of what we can do for the in memory DB and the MySQL production DB. I was actually thinking about having a prod.conf with evolutions turned off, and then leaving the evolutions for the local DB testing. That way development is super easy for contributors, and the only time we need to manually muck around with the DB is when we are deploying to production (which seems safer anyways). Thoughts? (Happy to revise if you'd like)

Comment (ErrorsAndGlitches) [reply conf/application.conf]: I was thinking about this a little today. Is it possible to have a directory that contains the memory and mysql specific evolution scripts and during build, it copies the scripts into the directory that scala play uses when running evolutions?

Using an in-memory database for local testing is super slick and then switching to mysql for prod or perhaps more end-2-end testing locally is such a nice feature to have.

**Comment by joshmcfarlane** [comment]
Discarding as this was solved by https://github.com/ColectivaLegal/ScalaSmsAlertSystem/pull/37 instead

**Pull request closed without merging**